docs/guide/vscode-getting-started.md

```diff
@@ -0,0 +1,97 @@
+# Getting Started in VSCode (C, Lua and Localizations)
+
+A Sapiens VSCode extension is at your disposal in the VSCode extension market that sets up a project for you to write Lua, C (a combination of both) and localization mods with. It uses the wonderful [Sapiens cmake template by suppergerrie2](https://github.com/Sapiens-OSS/sapiens-cmake-template).
+
+## Prerequisites
+The VSCode extension and cmake template have been tested on Windows and Linux.
+
+Please ensure you have the following installed on your system:
+- [Git](https://git-scm.com/)
+- [CMake](https://cmake.org/)
+    - **Windows** users should additionally install [Visual C++ Build Tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019)
+    - **Linux** users should additionally install [x86_64-w64-mingw32-cmake](https://aur.archlinux.org/packages/mingw-w64-cmake)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Sapiens VSCode Extension](https://marketplace.visualstudio.com/items?itemName=Sapiens-OSS.sapiens-vscode-extension)
+
+Ensure that `steam` is a recognized command on your system. If not, edit your PATH system environment variable to point to your Steam binary.
+- On **Windows**, your Steam binary is likely located at `C:\Program Files (x86)`, which is not by default in your PATH.
+
+Please take note of your Sapiens installation's mod directory:
+- For **Windows** users, this is by default installed at `C:\Users\<your_username>\AppData\majicjungle\sapiens\mods`.
```

Comment:
```suggestion
- For **Windows** users, this is by default installed at `%AppData%\majicjungle\sapiens\mods`.
```
You can use `%AppData%` to get into the Roaming folder. 
The path posted here is incorrect as it's missing Roaming in the path

Reply:
Let me test if using `%AppData%` works with Node.JS's filesystem first...

Reply:
Ok, that surprisingly enough works. Will add it